pyproject.toml

```diff
@@ -16,7 +16,7 @@ description = "Bootstrap-based Sphinx theme from the PyData community"
 dynamic = ["version"]
 readme = "README.md"
 
-requires-python = ">=3.5"
+requires-python = ">=3.6"
```

Comment:
Bump to 3.7+?

3.6 goes EoL in ~30 days. 🤷🏽

Reply:
Yep good idea

Reply:
We actually also don't test it